src/pages/landingPages/donate/components/testimonialSection/testimonialSection.js

```diff
@@ -8,17 +9,29 @@ const TestimonialSection = () => {
 				<div className="testimonial-content">
 					<div className="testimonial-form-content">
 						<h4 className="donate-title">Make An Impact</h4>
-						<p>
+						<p className="custom-text--large font-semibold">
 							To help us grow these opportunities, please consider
 							making <br />a tax-deductible donation to SpaceLab
 							today.
 						</p>
-						<button className="donate-button">Donate</button>
+						<Link
+							to={"/campaigns/donate/donate-payment"}
+							style={{ textDecoration: "none" }}
+						>
+							<button
+								className="donate-button"
+								component={Link}
+								to={"/campaigns/donate/donate-payment"}
+							>
+								Donate
+							</button>
+						</Link>{" "}
 					</div>
 					<div className="testimonial-data">
 						{SPACELAB_TESTIMONIALS.map(
 							({ name, feedback, imagePath }) => {
-								const image = require("../assets/" + imagePath);
+								const image = require("../../assets/" +
+									imagePath);
```

Comment:
This should be made into a separate component called TestimonialCard or something similar